src/main/java/org/zone/region/ZoneManager.java

```diff
@@ -280,4 +307,29 @@ public synchronized File save(Zone zone) throws ConfigurateException {
         loader.save(node);
         return file;
     }
+
+    public File zonesReload() {
+        File zonesFolder = new File("config/zone/zones/");
```

Comment:
You may want to clear the currently loaded zones before loading in the zones again

Reply:
umm how

Reply:
k